### DISCUSSION TITLE
Week 01 Check-in

### DISCUSSION THREAD

**Discussion by eculler** [earthlab-education/Earth-Analytics-2023-01-Intro#6] [Debriefs]
Please respond with W3-style debrief:

- **What**...did you learn this week?
- **So what?** Why was it important?
- **Now what?** What are you still curious about?

**Comment by ellenalamont17** [comment]
Starting off knowing nothing about GitHub nor how to navigate it, I struggled with just trying to find the basic help materials so I could help myself not be so confused. I also struggled in figuring out my way around the various repositories. I am feeling a bit better now that I have done some of the exercises and stalked the stack exchange. I think I am still having issues with jargon related to what to call files, folders, and the various components of the GitHub structure and would maybe benefit from a short explanation. Also, I still don't quite understand the concept of a pull. I get it is a request to check code/content, but I guess I don't understand how we interact with it, access recommendations, and whether to close of keep open. Thanks for a good first week.
